com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/BundleHelper.java

```diff
@@ -965,7 +965,9 @@ public static void moveBundleIfNeed(Project project, File bundleDir) throws IOEx
         if (publisher != null && publisher.shouldBeMovedIntoBundleFolder() && publisher instanceof ZipPublisher) {
             ZipPublisher zipPublisher = (ZipPublisher) publisher;
             File zipFile = zipPublisher.getZipFile();
-            Files.move(zipFile.toPath(), (new File(bundleDir, zipFile.getName())).toPath(), StandardCopyOption.REPLACE_EXISTING);
+            if (zipFile != null && zipFile.exists()) {
```

Comment:
When this is a build triggered immediately after the previous build without any changes, the archive will not exist. Not sure if it's ok to throw an error in such case? I decided to handle it, but such build will not contain lu archive.